app/src/main/java/space/taran/arknavigator/mvp/model/repo/index/AggregatedResourcesIndex.kt

```diff
@@ -7,6 +7,8 @@ class AggregatedResourcesIndex(
     private val shards: Collection<PlainResourcesIndex>)
```

Comment:
maybe add some kind of documentation so when other people read the code can know what this class is actually for?

Reply:
@darkovski88 yeah, we need to document core classes.
Maybe this will be what we add:
```
`AggregatedResourceIndex` is useful for "aggregated" navigation mode — a mode in which we navigate
through multiple indexed folders (roots). For a single-root navigation `PlainResourceIndex` can be used, and
both `AggregatedResourceIndex` and `PlainResourceIndex` should be transparently interchangeable,
meaning that both implementations support the same methods and they should not be used
otherwise as by `ResourceIndex` interface. Any component using `ResourceIndex` should
function in the same fashion independent on implementation of the index,
and only capability to look into multiple roots is gained
by passing `AggregatedResourceIndex` into the component. 
```
I am not sure if it's too wordy or not, I hope it explains the purpose of this class.